### DISCUSSION TITLE
error: Requested operation is not valid: cannot delete inactive domain with 1 checkpoint

### DISCUSSION THREAD

**Discussion by linuxgokan** [abbbi/virtnbdbackup#143] [Q&A / Support] [answered]
Hello

I get this error when deleting the virtual machine.
error: Requested operation is not valid: cannot delete inactive domain with 1 checkpoint

version: 1.9.23

How can I delete checkpoint?

**Comment by abbbi** [comment, marked as answer]
i dont know how this would be a bug in virtnbdbackup. Please check libvirt documentations. 
You either have to remove the checkpoints while the virtual machine is running before deleting or use virsh with the metadata option.
